idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -225,9 +225,7 @@ export default PageView.extend({
         { value: 'notuseful', title: 'feedbackUninstallAnswerNotUseful' },
         { value: 'other', title: 'feedbackUninstallAnswerOther' }
       ],
-      onSubmit: () => {
-        this.uninstall(this.updateAddon, this.model);
-      }
+      onSubmit: () => this.uninstall(evt)
```

Comment:
No, `evt` is a reference to [the parameter to `renderUninstallSurvey`](https://github.com/mozilla/idea-town/pull/377/files#diff-9a1984ab9661edeed8d9b7bed2951b8cR216)